src/command_modules/azure-cli-cosmosdb/HISTORY.rst

```diff
@@ -2,6 +2,12 @@
 
 Release History
 ===============
+
+unreleased
+^^^^^^^^^^^^^^^^^^
```

Comment:
fixed here:
e7ff5226888890f42a03b4a2fd8befdf4879e8b3

removed some local files which got in by mistake:
d02a54aa96b25b6c598a5068814ba7243280416f
79324e0ba3f37712be12e39a7f540bc355e1f951